src/sidebar/components/test/AnnotationView-test.tsx

```diff
@@ -1,26 +1,43 @@
 import { mockImportedComponents, waitFor } from '@hypothesis/frontend-testing';
 import { mount } from 'enzyme';
+import sinon from 'sinon';
+import type { SinonStub } from 'sinon';
 
-import AnnotationView, { $imports } from '../AnnotationView';
+import { getImports } from '../../../test-util/mockable-imports';
+import type { LoadAnnotationsService } from '../../services/load-annotations';
+import type { SidebarStore } from '../../store';
+import * as AnnotationViewExports from '../AnnotationView';
+import type { AnnotationViewProps } from '../AnnotationView';
 
 describe('AnnotationView', () => {
-  let fakeStore;
-  let fakeOnLogin;
+  const AnnotationView = AnnotationViewExports.default;
+  const $imports = getImports(AnnotationViewExports);
+
+  let highlightAnnotations: SinonStub;
+  let setExpanded: SinonStub;
+  let profile: SinonStub;
+  let fakeStore: SidebarStore;
+  let fakeOnLogin: SinonStub;
   let fakeUseRootThread;
-  let fakeLoadAnnotationsService;
+  let loadThread: SinonStub;
+  let fakeLoadAnnotationsService: LoadAnnotationsService;
```

Comment:
It is incorrect to use the real service as the type, because we definitely be expecting to access properties in tests that don't exist on the real service.

Reply:
Accessing the method stubs separately is the workaround for that, but it would be definitely nice to have a type that knows what's the shape of the mocked service/object, while it also knows which methods are sinon stubs.